modules/speech-to-text-workflowoperation/src/main/java/org/opencastproject/workflow/handler/speechtotext/SpeechToTextWorkflowOperationHandler.java

```diff
@@ -125,20 +129,53 @@ public WorkflowOperationResult start(WorkflowInstance workflowInstance, JobConte
           throws WorkflowOperationException {
 
     MediaPackage mediaPackage = workflowInstance.getMediaPackage();
+    WorkflowOperationInstance operation = workflowInstance.getCurrentOperation();
```

Comment:
```suggestion

```
Unused

Reply:
Thanks for noting that, initially I was to use that variable, but later I changed my idea.

It surprises me that Check Style didn't warn me about.